packages/components/src/utils/colors-values.js

```diff
@@ -137,9 +137,9 @@ export const ALERT = {
 export const UI = {
 	background: BASE.white,
 	backgroundDisabled: LIGHT_GRAY[ 200 ],
-	brand: BLUE.wordpress[ 700 ],
+	brand: `var( --wp-admin-theme-color, ${ BLUE.wordpress[ 700 ] })`,
```

Comment:
I'm wondering if we should actually  keep then aming `adminThemeColor` instead of `brand` and  `adminThemeColorDarker10` for the  "borderFocus" one? What do you think?

Reply:
I think swapping `brand` to `adminThemeColor` is okay. Or maybe just `themeColor` 👍 

However, I think keeping the other names like `border` or `borderFocus` is important, as it can be understood without having prior knowledge (in this case, knowledge of the naming and availability of the `--wp-admin-theme` color variant)

Reply:
> However, I think keeping the other names like border or borderFocus is important, as it can be understood without having prior knowledge (in this case, knowledge of the naming and availability of the --wp-admin-theme color variant)

I actually think we need both names (like we do in SASS), basically this:

```
UI.borderFocus = UI.adminThemeColorDarker10;
```

The colors variables are important because they are what defines a "theme".
And the border variables are important because they are what defines the usage.

Reply:
Ooo. Valid point. I can do that in a follow up PR